test/testhelper/utils.go

```diff
@@ -162,6 +162,14 @@ func GetFileDataLocally(filePath string) string {
 	return result
 }
 
+// GetFileDataRemote get file data from remote
+func GetFileDataRemote(sshClient *SSHClient, filePath string) []byte {
```

Comment:
s/GetFileDataRemote/GetRemoteFileData

the result is FileData, not Remote, right?